### ISSUE TITLE
Login failed

### ISSUE THREAD

**Issue by jessieflorrs** [rarcega/instagram-scraper#438]
Is anyone else getting an "Login failed" Error?
This is accompanied by another error to verify account along with a link.

Even after verifying my account again, trying to rerun the program trips this error again, which yields an "unusual login attempt" on the instagram site. 

I just saw this today so I'm not sure if it has happened to anyone else

**Comment by ingtellect** [comment]
I am getting the same error from today. Until yesterday it was fine.

**Comment by moodiest** [comment]
Am also encountering this issue as of last night.

**Comment by newty505** [comment]
Yep. Seems to be an issue after an update to the Instagram Login API - They've added an updated 'Unusual Login Attempt' segment which I'm guessing the current login method on the scraper  isn't using.

**Comment by thewracl** [comment]
Well, I can't confirm that and I consider myself a heavy user but I have no doubts you have the problems. So it seems IG rolls out the API changes in waves. Either per country/region or age of user-account. I have seen both cases. But I have seen a lot more 429 errors as it seems they have dropped the limit for API calls drastically. But that is all just user-based, not on a IP

**Comment by Baemir** [comment]
Have been having this problem for a few days now. Nothing I can do concerning authentication seems to get me past it.

**Comment by necrogami** [comment]
Any ways around this? I'm getting this every time i try and start the script.

**Comment by replay111** [comment]
Hi,

I am also always given this error:

```bash
ERROR: Login failed for <my account>
ERROR: Please verify your account at https://www.instagram.com/challenge/*
```
I've change password to be sure that I am providing the right one :(

**Comment by Eisregen04** [comment]
hey hey, i have the same issue on the script it wants to verify and i can use sms or email but there is no code send to me. on other devices it sends me to the login help page

**Comment by rikkiran** [comment]
Same problem here. Even after I verify it's me, it still gives me the suspicious login error whenever I try to run it. It's been going on for well over a month now. Anyone find a resolution to this?

**Comment by Baemir** [comment]
After trying all sorts of things, I became convinced that IG had effectively 'shadowbanned' my account, and that there would be no way to access it again no matter what I tried. Maybe I'm just paranoid, but I'm pretty sure it was gone for good and they just didn't want to tell me. So I tried making a new one, and that didn't work either; any new accounts I made were apparently created on their database but then instantly auto-banned, probably owing to the MAC address of the device I was using (I had changed my IP). Before trying to edit my MAC address, I thought I'd try to register an account on my phone first, and surely enough, that worked. I'm not going to use instagram-scraper again, I suggest looking up 'pyinstastories' and 'instalooter' here on github, they do the job and my new account is doing fine so far.

**Comment by rikkiran** [comment]
> > Same problem here. Even after I verify it's me, it still gives me the suspicious login error whenever I try to run it. It's been going on for well over a month now. Anyone find a resolution to this?
> 
> After trying all sorts of things, I became convinced that IG had effectively 'shadowbanned' my account, and that there would be no way to access it again no matter what I tried. Maybe I'm just paranoid, but I'm pretty sure it was gone for good and they just didn't want to tell me. So I tried making a new one, and that didn't work either; any new accounts I made were apparently created on their database but then instantly auto-banned, probably owing to the MAC address of the device I was using (I had changed my IP). Before trying to edit my MAC address, I thought I'd try to register an account on my phone first, and surely enough, that worked. I'm not going to use instagram-scraper again, I suggest looking up 'pyinstastories' and 'instalooter' here on github, they do the job and my new account is doing fine so far.

Thing is, I can access my account fine through the app and website. Even the Story Saver app works without issue. It's just this one.  I wish there was a fix as I really like this one, but I'll give the others a shot. Thanks.

**Comment by rikkiran** [comment]
Hey all, so I followed the guide that was posted here and it immediately worked! Changing the user agent and app did the trick.
https://github.com/rarcega/instagram-scraper/issues/259

**Comment by jessieflorrs** [comment]
Changing user agent in constants.py also worked for me. Hopefully this fixes it at least for a while
